test/2d.jl

```diff
@@ -119,6 +119,11 @@ using Rotations, StaticArrays, Base.Test
         show(io, MIME("text/plain"), r)
         str = String(take!(io))
         @test startswith(str, "2×2 RotMatrix{Float64}:")
+
+        rxyz = RotXYZ(1.0, 2.0, 3.0)
```

Comment:
This isn't a 2D rotation (just an odd place to put this text)

Reply:
Good call. I'll move this whole testset somewhere else.